UniConnectApp/app/src/test/java/com/example/uniconnect/UniConnectUnitTest.kt

```diff
@@ -73,12 +74,12 @@ class UniConnectUnitTest {
     @Test
     fun `Given service connects to University JSON stream when data are read and parsed then university collection should be greater than zero`() =
         runTest {
-        launch(Dispatchers.Main) {
-            givenUniversityServiceIsInitialized()
-            whenServiceDataAreReadAndParsed()
-            thenTheUniversityCollectionSizeShouldBeGreaterThanZero()
+            launch(Dispatchers.Main) {
+                givenUniversityServiceIsInitialized()
+                whenServiceDataAreReadAndParsed()
+                thenTheUniversityCollectionSizeShouldBeGreaterThanZero()
+            }
```

Comment:
automatic via kotlin style guide per https://kotlinlang.org/docs/coding-conventions.html#apply-the-style-guide